test_export_gh_config.yaml

```diff
@@ -0,0 +1,51 @@
+# Configuration file for standard GraphHopper routers.
+# 1 profile loaded with default speeds is defined for each vehicle type
+
+graphhopper:
+  datareader.file: ./test-data/micro_nor_cal.osm.pbf
+  gtfs.max_transfer_interpolation_walk_time_seconds: 300
+  graph.location: transit_data/export_test/graphhopper
```

Comment:
Of note is that this graph folder is different that the folder used for other unit tests. This is because the nationwide export command will actually build graph files, but we don't want those to be used by the other unit tests; we want to test the `import` command from a fresh start, essentially.